app/models/setting/remote_execution.rb

```diff
@@ -12,54 +13,63 @@ def self.load_defaults
       [
         self.set('remote_execution_fallback_proxy',
                  N_('Search the host for any proxy with Remote Execution, useful when the host has no subnet or the subnet does not have an execution proxy'),
-                 false),
+                 false,
+                 N_('Fallback Any Proxy')),
```

Comment:
This feels a bit odd compared to `Fallback Without Proxy`, maybe something like `Fallback to Any Proxy` would be better?

Reply:
Sounds better indeed, my brain was a bit fried after writing all these descriptions :laughing: 

@adamruzicka Can you do that change on merge? Just clone the branch, make the change, add it, `git commit --amend` it , cherry-pick that change to master and push. It's just so we can save another 2h long Jenkins job

Reply:
No, problem, will do